### PR TITLE
[entropy complex/security] countermeasure lists and labels updated

### PR DESCRIPTION
**Pull request by mwbranstad**

### CODE REVIEW

File: hw/ip/entropy_src/data/entropy_src.hjson [hunk 1]
```diff
@@ -82,16 +82,28 @@
...
+      desc: "Registers have multi-bit encoded fields."
+    }
+    { name: "CONFIG.REDUN"
+      desc: "Threshold register has an inverted copy to compare against."
+    }
     { name: "FSM.SPARSE"
       desc: "Sparse state machine implementation."
     }
+    { name: "RNG.BKGN_CHK"
+      desc: "Random number generater is protected with continuous background health checks."
```
Comment (cdgori) [inline hw/ip/entropy_src/data/entropy_src.hjson:98]: nit: generater -> generator

File: hw/ip/csrng/data/csrng.hjson [hunk 2]
```diff
@@ -86,16 +86,22 @@
...
+      desc: "Registers have multi-bit encoded fields."
+    }
     { name: "FSM.SPARSE"
       desc: "Sparse state machine implementation."
     }
     { name: "CTR.REDUN"
       desc: "Counter hardening for generate command counter."
     }
-    { name: "LOGIC.INTEGRITY"
+    { name: "SW_GENBITS.BUS.INTEGRITY"
```
Comment (cdgori) [inline hw/ip/csrng/data/csrng.hjson:101]: INTEGRITY -> CONSISTENCY for these consecutive values checks

File: hw/ip/edn/data/edn.hjson [hunk 3]
```diff
@@ -51,16 +51,22 @@
...
+      desc: "Registers have multi-bit encoded fields."
+    }
     { name: "FSM.SPARSE"
       desc: "Sparse state machine implementation."
     }
     { name: "CTR.REDUN"
       desc: "Counter hardening on the generate command maximum requests counter."
     }
-    { name: "LOGIC.INTEGRITY"
+    { name: "CS_RDATA.BUS.INTEGRITY"
```
Comment (cdgori) [inline hw/ip/edn/data/edn.hjson:66]: INTEGRITY -> CONSISTENCY for these consecutive values checks

File: hw/ip/entropy_src/data/entropy_src.hjson [hunk 4]
```diff
@@ -82,16 +82,28 @@
...
+    }
     { name: "CTR.REDUN"
       desc: "Counter hardening for all health test counters."
     }
-    { name: "LOGIC.INTEGRITY"
+    { name: "ESFINAL_RDATA.BUS.INTEGRITY"
```
Comment (cdgori) [inline hw/ip/entropy_src/data/entropy_src.hjson:103]: INTEGRITY -> CONSISTENCY for these consecutive values checks

**Comment by cdgori** [comment]
Please adjust per comments above.  You'll also need to add another reviewer with commit rights or this will be stuck.  It won't even let me start a "review" because of the missing committer.

**Comment by tjaychen** [comment]
this looks good to me.

**Review by tjaychen — APPROVED** [review]

**Pull request merged**